### DISCUSSION TITLE
EntryType as argument for MealPlan API Endpoint

### DISCUSSION THREAD

**Discussion by Siege36** [mealie-recipes/mealie#1112] [Feature Request]
**Before submitting this feature request I have**
_COMPLETE AND FILL THESE OUT_
- [x] Reviewed the documentation and verified that my feature request isn't already a feature.
- [x] Reviewed the existing feature requests and my feature has not been requested.
- [x] Checked the [tasks taged](https://github.com/hay-kot/mealie/issues?q=is%3Aissue+is%3Aopen+label%3Atask+) issues and verified my feature is not covered

**Please Describe The Problem To Be Solved**
I wish there was a way to pass a argument entryType to the mealplan api. That way I could set up different sesnors in home assistant. For example: dinner tonight, lunch today, etc. without hving to do work in the receiving app.

**(Optional): Suggest A Solution**
/api/groups/mealplans/today/entryType/{entryType} 

If there are multiple solutions, please present each one separately. Save comparisons for the very end.)

**Additional Information**
_FILL THESE OUT_
- [ ] If the feature is accepted I'm willing to submit a PR to provide this feature

**Comment by michael-genson** [comment]
The "Get All" endpoint allows arbitrary filters, so you can accomplish this now. [More on the filter API here](https://docs.mealie.io/documentation/getting-started/api-usage/#filtering).

You can't filter on the "today" endpoint, but you can use the "get all" endpoint with a date filter, which is effectively the same thing.
